packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -167,5 +167,5 @@ function toPerformanceEntryRepresentation(entry: RumPerformanceEntry): Performan
  * @returns number between 0 and 1 which represents tracing sample rate
  */
 function getRulePsr(configuration: RumConfiguration) {
-  return configuration.tracingSampleRate / 100
+  return configuration.tracingSampleRate ? configuration.tracingSampleRate / 100 : undefined
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏a tracing sample rate set to `0` would return `undefined`

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** is tracing sample rate a valid input?

Reply:
https://docs.datadoghq.com/real_user_monitoring/browser/#initialization-parameters
<img width="744" alt="Capture d’écran 2022-08-26 à 10 06 35" src="https://user-images.githubusercontent.com/1331991/186854273-65992b85-6dd0-4821-8c08-b5f10f469f3f.png">